test_util/test_upgrade_vpc.py

```diff
@@ -247,7 +247,7 @@ def setup_cluster_workload(self, dcos_api: DcosApiSession, healthcheck_app: dict
             # the master's view after the upgrade.
             # See this issue for why we check for a difference:
             # https://issues.apache.org/jira/browse/MESOS-1718
-            self.task_state_start = self.get_master_task_state(dcos_api, self.tasks_start[0])
+            self.task_state_start = self.get_master_task_state(dcos_api, self.tasks_start[self.test_app_ids[0]])
```

Comment:
Yes, consistency is the goal here.  We can test more than one task, but we also don't need to test more than one task; to confirm that each tasks' state has not changed between upgrades.